modules/markdown/markdown.tsx

```diff
@@ -37,7 +45,7 @@ export type MarkdownProps = {
 		CodeBlock?: StyleProp<TextStyle>
 		Emph?: StyleProp<TextStyle>
 		Heading?: StyleProp<TextStyle>
-		Image?: StyleProp<ViewStyle>
+		Image?: StyleProp<ImageStyle>
```

Comment:
While this is "more correct" now… let's not try actually rendering anything, I'm pretty sure that's broken. (The props of img to Image are pretty different)